### PR TITLE
perf: ws server for local discord UI dev

### PR DESCRIPTION
**Pull request by ryanemcdaniel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +32,7 @@
     "prepare": "husky"
   },
   "dependencies": {
+    "@aws-sdk/client-apigatewaymanagementapi": "3.699.0",
```
Comment (ryanemcdaniel) [inline package.json:35]: planning to contribute to `@effect-aws` to normalize AWS SDK usages. decent stop-gap for now

**Review by martinaa23 — COMMENTED** [review]
good working ws

File: dev/ws-proxy.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+const wss = await pipe(
+    CFG.redacted('/dffp/qual/dev_ws'),
+    E.provide(L.setConfigProvider(fromParameterStore())),
+    E.runPromise,
+);
+
+
+const socket = new WebSocket(RDT.value(wss));
```
Comment (ryanemcdaniel) [inline dev/ws-proxy.ts:16]: discord requires a single stable URL to forward slash command / button press inputs...

this socket connects to our API lambda and overrides further routing across backend to the local dev connection

this backdoor is only available for qual and supports only a single active connection across all devs = only 1 of us at a time can locally dev UI components.

it's possible to extend this to multiple ppl at once, maybe if we get a larger/more active dev team

File: dev/ws-proxy.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+const server = hapi.server({
+    host: 'localhost',
+    port: 3000,
+});
+
+await server.register(Nes);
+
+server.subscription('/dev');
```
Comment (ryanemcdaniel) [inline dev/ws-proxy.ts:52]: this Hapi server https://hapi.dev keeps a stable connection to the APIGW lambdas and forwards messages to the local dev client

this keeps # of connect/disconnect down to APIGW while continuously rebuilding code for local changes

**Review by martinaa23 — APPROVED** [review]
good improvement

**Pull request merged**